test/codecs.test.js

```diff
@@ -110,8 +130,7 @@ QUnit.test('parses video only codec string', function(assert) {
     {
       codecCount: 1,
       videoCodec: 'avc1',
-      videoObjectTypeIndicator: '.42001e',
-      audioProfile: null
```

Comment:
We used to return null for audioProfile on every return, but we really shouldn't